config/initializers/rails_admin.rb

```diff
@@ -152,8 +152,16 @@
       end
     end
     edit do
-      field :uri
-      field :name
+      field :uri do
+        read_only do
+          true unless bindings[:view]._current_user.role == 'admin'
```

Comment:
If we really want per-attribute permissions, then they should be declared in `Ability`, with custom methods if necessary. not here.